README.md

```diff
@@ -8,9 +8,10 @@ Swinject
 [![Travis CI](https://travis-ci.org/Swinject/Swinject.svg?branch=master)](https://travis-ci.org/Swinject/Swinject)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![CocoaPods Version](https://img.shields.io/cocoapods/v/Swinject.svg?style=flat)](http://cocoapods.org/pods/Swinject)
+[![Swift PM compatible](https://img.shields.io/badge/SwiftPM-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
```

Comment:
Thank you for your careful eyes. For another reason, I dropped the Swift PM badge at its following commit (25934ab306680214213380b258109f2f4a0ef24c) to fit all badges in a line.